src/shadowbox/model/access_key.ts

```diff
@@ -27,19 +27,19 @@ export interface ProxyParams {
   readonly password: string;
 }
 
-// Parameters needed to limit access key data usage over a sliding window.
-export interface AccessKeyQuota {
+// Parameters needed to limit access key data usage over a sliding timeframe.
+export interface AccessKeyLimit {
```

Comment:
Should we use `DataLimit` instead of only `Limit` here and throughout the code?
We could have other types of limits: number of devices, sites that one can access, whether they can share.

Reply:
Done.